frontend/src/app/utils/internalChartHelper.ts

```diff
@@ -398,6 +398,48 @@ function getMeta(key, column) {
   };
 }
 
+export function determineTheMergeChartStyleConfigsValueType(
+  tEle: any,
```

Comment:
if input parameters only use the `value`, it is better to pass the value directly, not the whole object?